src/custom/components/TransactionSettings/TransactionSetttingsMod.tsx

```diff
@@ -233,7 +233,7 @@ export default function SlippageTabs({
               ? 'Enter a valid slippage percentage'
               : slippageError === SlippageError.RiskyLow
               ? 'Your transaction may expire'
-              : 'Your transaction may be frontrun'}
+              : 'High slippage amount selected'}
```

Comment:
nanonit: maybe just "slippage" instead of "slippage amount" (it's technically a %)